content/software/plc-ide/tutorials/plc-programming-introduction/content.md

```diff
@@ -100,10 +104,32 @@ Inside the **Tool window** you can drag and drop variables, or click the **Inser
 
 ![Watch a variable](assets/watch.png)
 
+### Library Management
+
+The PLC IDE manages the library add-on and usage differently. To add the desired library, go to **Resources** tab and click the **Libraries** listed under the **Sketch**.
+
```

Comment:
This is my proposal to have a better user experience on this little section:

So, we have the line that defines the "action to do"

Now we put just below the animation/gif that shows that physical action on the software:

```suggestion
![Adding a library to the PLC IDE Arduino Sketch](assets/plcide-library-management.gif)
```
And after it we have the line that explains the second action but it also explained in the animation from above (so both actions explained in the text from top and bottom are linked and explained in the animation too)

After it we have the rest of the text and paragraphs putting more detail.

Please follow this architecture please.

This way we can remove the original image

Reply:
okay, I'll apply the change and check to refine it if necessary